src/scripts.js

```diff
@@ -26,18 +26,48 @@ const userInfoBody = document.getElementById('userInfoBody');
 const greeting = document.getElementById('helloUser');
 const stepGoal = document.getElementById('stepGoal');
 
-const userRepo = new UserRepository(userData);
-const randomId = Math.floor(Math.random() * userRepo.usersData.length) + 1;
-const randomUser = userRepo.findUser(randomId);
+// const userRepo = new UserRepository(userData);
+// const randomId = Math.floor(Math.random() * userRepo.usersData.length) + 1;
+// const randomUser = userRepo.findUser(randomId);
 
```

Comment:
I commented out all previous code that may cause error messages.. the goal of this PR is to just showcase what is happening with the fetch and promise all